src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Buttons } from '../../constants/common.constants';
+import classes from './SearchBar.module.scss';
+
+interface ISearchBarProps {
+  text: string;
+  onTextChange: (value: string) => void;
+}
+
+class SearchBar extends React.Component<ISearchBarProps> {
+  private handleTextChange: (e: React.ChangeEvent<HTMLInputElement>) => void = (
+    e: React.ChangeEvent<HTMLInputElement>
```

Comment:
u can use or first type or the second type defenition

Reply:
really)